### PR TITLE
Bug fix - empty tags are breaking original metadata

### PR DESCRIPTION
**Pull request by mberacochea**
Example of query that doesn't work ATM:
```
$ mg-toolkit original_metadata -a ERP012177
File "/rds/project/rds-aFEMMKDjWlo/software/mambaforge-pypy3/envs/ena-metadata/lib/python3.9/site-packages/mg_toolkit/metadata.py", line 151, in fetch_metadata
    _meta = self.get_metadata(sample["sample_accession"])
  File "/rds/project/rds-aFEMMKDjWlo/software/mambaforge-pypy3/envs/ena-metadata/lib/python3.9/site-packages/mg_toolkit/metadata.py", line 83, in get_metadata
    key_value = value.text.strip()
AttributeError: 'NoneType' object has no attribute 'strip'
```

### CODE REVIEW

File: mg_toolkit/metadata.py [hunk 1]
```diff
@@ -71,17 +69,21 @@ def get_metadata(self, sample_accession):
...
-            if value is not None:
-                key_value = value.text.strip()
-            if units is not None:
+            if not value:
+                key_value = ""
+            else:
+                key_value = value.text.strip() if value.text else ""
+
+            if units and units.text:
                 key_value += units.text.strip()
```
Comment (SandyRogers) [inline mg_toolkit/metadata.py]:
```suggestion
            key_value = value.text.strip() if value and value.text else ""
            if units and units.text:
                key_value += units.text.strip()
```

File: tests/integration/original_metadata/ERP001178.csv.md5 [hunk 2]
```diff
@@ -1 +1 @@
-102059fc508c6c3f950ee62dbb3cbc72  ERP001178.csv
+7b26ba440c42b3f8b9b41c378b5a36be  ERP001178.csv
```
Comment (SandyRogers) [inline tests/integration/original_metadata/ERP001178.csv.md5:1]: how did the md5 change if the csv fixture hasn't changed?

**Review by SandyRogers — COMMENTED** [review]
Cool – thanks for working on this Martin! Couple of little things inline.

File: tests/integration/original_metadata/ERP001178.csv.md5 (hunk 2 above)
Comment (mberacochea) [reply tests/integration/original_metadata/ERP001178.csv.md5:1]: there is something funky going one

Comment (mberacochea) [reply tests/integration/original_metadata/ERP001178.csv.md5:1]: The column order was causing issues.

**Pull request merged**